backend/Models/Database/Locations.cs

```diff
@@ -0,0 +1,9 @@
+namespace WhaleSpotting.Models.Database
+{
+    public class Locations
+    {
+        public int Id { get; set; }
+        public string Description { get; set; }
```

Comment:
As in `WhaleSpecies.cs`, you must add a list of WhaleSpecies here to allow Entity Framework to set up the join table:
```suggestion
        public string Description { get; set; }
        public List<WhaleSpecies> WhaleSpecies { get; set; }
```